finish/system/pom.xml

```diff
@@ -35,12 +35,12 @@
         </dependency>
         <dependency>
             <groupId>io.openliberty.features</groupId>
-            <artifactId>mpRestClient-1.1</artifactId>
```

Comment:
The `mpRestClient` feature is not specified in the `server.xml` file. Is it being used?

Reply:
The test may require it

Reply:
addressed by #67